app/advancedfilters/advancedFilters.html

```diff
@@ -41,10 +41,10 @@
       </div>
     </li>
 
-    <!-- GENE PRODUCTS -->
+    <!-- GENE PRODUCT IDs -->
     <li uib-dropdown auto-close="outsideClick" is-open="status.isopenGP">
-      <button type="button" class="btn btn-default quickgo-btn icon" ng-class="{'icon-functional active-filter' : isActiveFilter('gpID') || isActiveFilter('gpType') || isActiveFilter('gpSet')}" data-icon="/" uib-dropdown-toggle>
-        Gene products <span class="caret"></span>
+      <button type="button" class="btn btn-default quickgo-btn icon" ng-class="{'icon-functional active-filter' : isActiveFilter('gpID') ||  isActiveFilter('gpSet')}" data-icon="/" uib-dropdown-toggle>
+        Gene productIDs <span class="caret"></span>
```

Comment:
Something changed regarding the expected output for ng-repeat in line 54 li ng-repeat="(gpID, value) in filters.gpID" class="checkbox">
I tried /annotations?gpID=T1DG24, because a filter is already applied I was expecting to see a selected checkbox with "T1DG24" as label. Not there. It appear though if I click in Gene ProductIDs, click out, and click in the filter again.